### ISSUE TITLE
[BUG] Groupby AttributeError

### ISSUE THREAD

**Issue by mrocklin** [rapidsai/cudf#534]
**Describe the bug**
Groupby aggregations don't seem to have the ability to select columns

**Steps/Code to reproduce bug**

```python
import pandas as pd, cudf
import numpy as np
pdf = pd.DataFrame({'x': [1, 2, 3, 4], 'y': [0, 1, 0, 1]})
gdf = cudf.DataFrame.from_pandas(pdf)
gdf.groupby('y').x.mean()

AttributeError: 'Groupby' object has no attribute 'x'
```

**Comment by mrocklin** [comment]
I'm adding this to this release.  I *think* it's critical?  I'm not sure though.   It would be useful to have a cudf maintainer take a look at the issue and see if I'm not doing something dumb here.

**Comment by mrocklin** [comment]
Looking at the test suite it seems that we only support the pattern 

```python
df.groupby(keys).agg(...)
```

Is this correct?  If so then we should probably add a NotImplementedError (or just solve the problem).  My sense is that this is one of the more (most?) common ways for users to start with groupby-aggregations.

**Comment by kkraus14** [comment]
I don't think this is critical for release and can be pushed to 0.6 as you can always do `.agg({'x': 'mean'})`. That being said we need to create a proper SeriesGroupBy object to allow this type of behavior.

**Comment by kkraus14** [comment]
@mrocklin is this critical path for Dask?

**Comment by mrocklin** [comment]
Not that I'm aware of.  I raised this as a cudf user, not a dask-cudf user.

**Comment by kkraus14** [comment]
Resolved by #878.
